lib/AST/ASTContext.cpp

```diff
@@ -908,6 +914,75 @@ FuncDecl *ASTContext::getEqualIntDecl() const {
   return nullptr;
 }
 
+FuncDecl *ASTContext::getMutatingXorIntDecl() const {
+  if (Impl.MutatingXorIntDecl)
+    return Impl.MutatingXorIntDecl;
+
+  auto intType = getIntDecl()->getDeclaredType();
+  auto inoutIntType = InOutType::get(intType);
+  SmallVector<ValueDecl *, 30> xorFuncs;
```

Comment:
Thanks for pointing that out—I didn't go back and change it after I cribbed the function from `==`. It looks like there are currently ~12, so I'll use 16.